src/Storefront/Resources/views/storefront/page/account/profile/index.html.twig

```diff
@@ -34,11 +34,18 @@
                                         {# @deprecated tag:v6.8.0 - Unused variable `showVatIdField` will be removed, use variable showCompanyFields instead #}
                                         {% set showVatIdField = true %}
 
+                                        {# If customer is logged in (on profile page), they should be able to freely change their account type #}
```

Comment:
@shopware/product-cc-b2b FYI, want to make sure that this wouldn't case mayhem on your side :smile: